setup.cfg

```diff
@@ -11,5 +11,16 @@ packages = ["nesta_ds_utils"]
 
 [options]
 python_requires = >=3.8
-install_requires = 
+install_requires =
+extras_requires = {
+    "dev": [
+      "Sphinx==5.2.3",
+      "sphinxcontrib-applehelp==1.0.2",
+      "sphinxcontrib-devhelp==1.0.2",
+      "sphinxcontrib-htmlhelp==2.0.0",
+      "sphinxcontrib-jsmath==1.0.1",
+      "sphinxcontrib-qthelp==1.0.3",
+      "sphinxcontrib-serializinghtml==1.1.5",
+      "pytest==7.1.3"
+        ]}
```

Comment:
Do you definitely need quotation marks for the requires? I think you need them if you include the requires in  `setup.py`, but not in `setup.cfg`. I might be wrong though! 

I was trying to install the package with dev requires via `pip install -e .[dev]` but it doesn't work. I wonder if it is because of that.

Reply:
Yep just fixed it you were right - no quotes, newlines, and semi-colons instead of commas.

Also, fwiw  I had to run `pip install -e ."[dev]" ` it didn't work without the quotes for some reason